sys/include/net/nanocoap_sock.h

```diff
@@ -240,6 +240,23 @@ ssize_t nanocoap_sock_put(nanocoap_sock_t *sock, const char *path,
                           const void *request, size_t len,
                           void *response, size_t len_max);
 
+/**
+ * @brief   Simple non-confirmable PUT
+ *
+ * @param[in]   sock    socket to use for the request
+ * @param[in]   path    remote path
+ * @param[in]   request buffer containing the payload
+ * @param[in]   len     length of the payload to send
+ * @param[out]  response buffer for the response, may be NULL
+ * @param[in]   len_max length of @p response
+ *
+ * @returns     length of response payload (or 0) on success
```

Comment:
```suggestion
 * @returns     length of response payload on success
 * @returns     0 if the request was sent and no response buffer was provided,
 *              independently of success (because no response is requested in that case)
```